Source/WebCore/rendering/style/RenderStyleSetters.h

```diff
@@ -471,7 +471,8 @@ inline void RenderStyle::setPseudoElementNameArgument(const AtomString& identifi
         || pseudoElementType() == PseudoId::ViewTransitionImagePair
         || pseudoElementType() == PseudoId::ViewTransitionNew
         || pseudoElementType() == PseudoId::ViewTransitionOld
-        || pseudoElementType() == PseudoId::Highlight);
+        || pseudoElementType() == PseudoId::Highlight
+        || identifier.isNull());
```

Comment:
Do we actually want to allow passing a null string to the SET_NESTED?

Reply:
I think that's fine as long as we're not setting it for a named pseudo-element